src/lib.rs

```diff
@@ -178,10 +187,26 @@ impl<T: Runtime> ClientBuilder<T> {
             runtime_version: runtime_version?,
             _marker: PhantomData,
             page_size: self.page_size.unwrap_or(10),
+            signed_options: ClientSignedOptions {
+                mortal_period: Some(DEFAULT_MORTAL_PERIOD),
+            },
         })
     }
 }
 
+/// Construction options for a signed extrinsic
+#[derive(Copy, Clone)]
+struct ClientSignedOptions {
+    /// The period, measured in blocks, that an extrinsic will live for, starting from a checkpoint
+    /// block.
+    ///
+    /// Rules for how the period will be encoded in an extrinsic:
+    /// `mortal_period == None`: immortal transaction
+    /// `0 <= mortal_period <= 65536`: rounded up to the closest power of 2, starting at 4
+    /// `65536 < mortal_period`: Min(65536, mortal_period)
```

Comment:
I think it is better to refer to https://docs.rs/sp-runtime/2.0.0/sp_runtime/generic/enum.Era.html#variant.Mortal

It's not obivious with the edge case when `BlockHashCount` is less than `mortality_period`